index.html

```diff
@@ -11,21 +11,43 @@
 
 <body>
     <div class="container">
-        <h1>Awesome books</h1>
-        <section id="books-list">
+        <nav>
+            <h2>Awesome books</h2>
+            <ul class="options">
+                <li class="border-right menu-option blue">List</li>
+                <li class="border-right menu-option">Add new</li>
+                <li class="menu-option">Contact</li>
+            </ul>
+        </nav>
+        <div class="main-section">
+            <h1 class="home-title">Awesome books</h1>
+            <section id="books-list" class="list-view">
 
-        </section>
-        <section class="books-crud">
-            <div class="indicator"></div>
-            <h2>Add a new book</h2>
-            <form method="post" id="book-form">
-                <input type="text" placeholder="Title" id="input-title" class="input-form" required />
-                <input type="text" placeholder="Author" id="input-author" class="input-form" required />
-                <input type="submit" value="Add" class="addNewBook" />
-            </form>
-        </section>
+            </section>
+            <section class="books-crud views hidden">
+                <h2>Add a new book</h2>
+                <form method="post" id="book-form">
+                    <input type="text" placeholder="Title" id="input-title" class="input-form" required />
+                    <input type="text" placeholder="Author" id="input-author" class="input-form" required />
+                    <input type="submit" value="Add" class="addNewBook" />
+                </form>
+            </section>
```

Comment:
[OPTIONAL] You should ensure that your design matches templates and wireframes as much as possible. I can see some layout differences but I am only letting you guys off because the aim of this project is to learn something new.